kinetic/distribution.yaml

```diff
@@ -2678,6 +2678,14 @@ repositories:
       url: https://github.com/naoki-mizuno/ds4_driver.git
       version: master
     status: maintained
+  duckbot-ros:
+    release:
+      packages:
+      - duckbot
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/sachinkum0009/duckbot-ros.git
```

Comment:
It looks like you have used the same repository for the upstream as well as the release repository. This is highly not recommeded. The release repositories snapshot the upstream repositories and create a lot of tags and branches that are distracting for development and significantly slow down development by significantly increasing the repository size each release. Please redo this following the instructions carefully about setting up a separate release repository.